client-js/app/scripts/app.ts

```diff
@@ -368,9 +400,23 @@ window.addEventListener('WebComponentsReady', () => {
       try {
         await attemptLogin(mgr, host, window.localStorage);
       } catch (e) {
-        await loginWithDialog(mgr, host);
+        if (topLevel) {
+          await loginWithDialog(mgr, host);
+        } else {
+          throw e;
+        }
       }
     }
+
+    var id = setInterval(() => {
```

Comment:
I just noticed this pinging being logged on the server side, what's the purpose of this? Is this to see if the manager is present? If so, doesn't the websocket keep track if it's open by itself?